_dev/src/components/pmax/pmax-integration-modals/pmax-discover-modal.vue

```diff
@@ -6,29 +6,30 @@
     v-bind="$attrs"
     @ok="confirm"
   >
-    <!-- it's temporary while waiting for the images -->
-    <div class="d-flex mb-4">
+    <div class="d-flex mb-3">
       <img
-        class="mr-4"
-        :src="content.img"
+        class="mr-4 img-fluid d-block"
+        :src="require(`@/assets/images/pmax-modal/${content.img}`)"
+        alt=""
```

Comment:
Don't forget the alt for accessibility

Reply:
I don't know what I can put in alt, do you have idea ?